packages/splits-sdk/src/client/data.ts

```diff
@@ -344,12 +346,20 @@ export class DataClient {
     }
 
     const internalBalances = formatAccountBalances(response.balances)
+    const warehouseBalances = formatAccountBalances(response.warehouseBalances)
     if (response.type === 'user') {
       // Only including split main balance for users
       return {
         withdrawn,
         distributed,
-        activeBalances: internalBalances,
+        activeBalances: mergeWith(
```

Comment:
hey @r0ohafza sorry i missed this pr.

is it ok to be merging these together? the caller then doesn't know if the balance is in splitmain or in the warehouse, right? does it make more sense to separate? or is there some easy way to tell?